main.html

```diff
@@ -40,22 +41,12 @@
         <div class="btn-group dropend w-100">
             <button class="btn btn-lg dropdown-toggle w-100 text-wrap" data-bs-toggle="dropdown" aria-expanded="false">Template Management</button>
             <div class="dropdown-menu">
-                <div id="newTemplate" title="Create a new template" class="btn btn-outline-primary w-100 my-1" onclick="createTemplate()">New Template</div>
+                <div id="newTemplate" title="Create a new template" class="btn btn-outline-primary w-100 my-1" onclick="createNewTemplate()">New Template</div>

```

Comment:
This works but I have to click the button twice so probably something with the js code isn't quite right